### PR TITLE
Bluetooth: Shell: Added mtu command

### PR DESCRIPTION
**Pull request by Thalley**
Added the att_mtu command to get the current MTU of
the default connection.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/shell/gatt.c [hunk 1]
```diff
@@ -1095,6 +1109,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(gatt_cmds,
 	SHELL_CMD_ARG(get, NULL, "<start handle> [end handle]", cmd_get, 2, 1),
 	SHELL_CMD_ARG(set, NULL, "<handle> [data...]", cmd_set, 2, 255),
 	SHELL_CMD_ARG(show-db, NULL, "[uuid] [num_matches]", cmd_show_db, 1, 2),
+	SHELL_CMD_ARG(att_mtu, NULL, "Output ATT MTU size", cmd_att_mtu, 1, 0),
```
Comment (Vudentz) [inline subsys/bluetooth/shell/gatt.c:1112]: Id use just mtu to make it short/easier to type.

Comment (Thalley) [reply subsys/bluetooth/shell/gatt.c]: Good idea, updated

**Review by joerchan — APPROVED** [review]

**Comment by joerchan** [comment]
@Thalley Needs to be rebased now.

**Comment by Thalley** [comment]
@joerchan Seems like the conflict was that the the change had already been done by someone else.

**Comment by joerchan** [comment]
Oh right. Up to you if you want to shorten the command or close the PR :)

**Comment by Thalley** [comment]
Closed as the command had already been implemented.

**Pull request closed without merging**